packages/core/contracts/oracle/implementation/VotingV2.sol

```diff
@@ -123,7 +123,7 @@ contract VotingV2 is Staker, OracleInterface, OracleAncillaryInterface, OracleGo
     bytes32[] public pendingPriceRequests;
 
     // Spam deletion requests. These are requests to delete pending price requests that are still to be voted on.
-    SpamDeletionRequest[] internal spamDeletionProposals;
+    SpamDeletionRequest[] public spamDeletionProposals;
```

Comment:
This can be public as we had a public getter function for this, that I have removed to save some bytecode